chsdi/views/wmtscapabilities.py

```diff
@@ -9,6 +9,37 @@
 from chsdi.lib.filters import *
 
 
+DEFAULT_TILEMATRIXSET = {0: [4000, 1, 1, 14285714.2857],
+                         1: [3750, 1, 1, 13392857.1429],
+                         2: [3500, 1, 1, 12500000.0],
+                         3: [3250, 1, 1, 11607142.8571],
+                         4: [3000, 1, 1, 10714285.7143],
+                         5: [2750, 1, 1, 9821428.57143],
+                         6: [2500, 1, 1, 8928571.42857],
+                         7: [2250, 1, 1, 8035714.28571],
+                         8: [2000, 1, 1, 7142857.14286],
+                         9: [1750, 2, 1, 6250000.0],
+                         10: [1500, 2, 1, 5357142.85714],
+                         11: [1250, 2, 1, 4464285.71429],
+                         12: [1000, 2, 2, 3571428.57143],
+                         13: [750, 3, 2, 2678571.42857],
+                         14: [650, 3, 2, 2321428.57143],
+                         15: [500, 4, 3, 1785714.28571],
+                         16: [250, 8, 5, 892857.142857],
+                         17: [100, 19, 13, 357142.857143],
+                         18: [50, 38, 25, 178571.428571],
+                         19: [20, 94, 63, 71428.5714286],
+                         20: [10, 188, 125, 35714.2857143],
+                         21: [5, 375, 250, 17857.1428571],
+                         22: [2.5, 750, 500, 8928.57142857],
+                         23: [2.0, 938, 625, 7142.85714286],
+                         24: [1.5, 1250, 834, 5357.14285714],
+                         25: [1.0, 1875, 1250, 3571.42857143],
+                         26: [0.5, 3750, 2500, 1785.71428571],
+                         27: [0.25, 7500, 5000, 892.857142857],
+                         28: [0.10, 18750, 12500, 357.142857143]}
```

Comment:
no, not yet...
but it would be a good idea to add the default tilematrixset to the bod.
what's the dpi for the scale column?